add section to prom operator instructions about necessary label (#8379)
grafana/loki: docs/sources/installation/helm/monitor-and-alert/index.md

        targetLabel: cluster
```

> In order to make sure the Prometheus Operator discovers the `ServiceMonitor` resources deployed by the `loki` chart, you will need to make sure those resources have the correct labels the Prometheus Operator is configured to look for. By default this is the key value label pair `release: prometheus`. Make sure the Loki `ServiceMonitor`s have this label by adding the following to the `values.yaml` for your Loki helm chart:

Reviewer: ```suggestion
In order for the Prometheus Operator to discover the `ServiceMonitor` resources deployed by the `loki` chart, those resources must have the labels the Prometheus Operator is configured to expect. By default this is the key value label pair `release: prometheus`. Make sure the Loki `ServiceMonitor`s have this label by adding the following to the `values.yaml` for your Loki Helm chart:
```

```yaml
monitoring:
  serviceMonitor:
    labels:

>       release: prometheus

Reviewer 2: unfortunately the helm-chart I used to install the prom operator used a different labelset

```
release: prom-stack
```

Maybe we can include instructions on how to figure out what label is needed?

Author: @slim-bean which helm chart did you use? did you not use `kube-prometheus-stack`?

Author: I think we should keep the instructions on `kube-prometheus-stack` as that install `kube-state-metrics` with the correct recodring rules, `node-exporter`, and the Prometheus Operator. One stop shop for everything someone needs to enable self-monitoring.

```

> This is also true for the `PrometheusRule` resource deployed by the Helm chart, which in addition to a label, need to be in the same namespace as the Prometheus Operator. For example, if you installed the Prometheus Operator in the `monitoring` namespace, you would need to also add the following to the `values.yaml` for you Loki helm chart to ensure recroding rules are properly discoverd:

Reviewer: ```suggestion
This is also true for the `PrometheusRule` resource deployed by the Helm chart, which in addition to a label, need to be in the same namespace as the Prometheus Operator. For example, if you installed the Prometheus Operator in the `monitoring` namespace, you would need to add the following to the `values.yaml` for your Loki Helm chart to ensure recording rules are properly discovered:
```
